bottles/frontend/ui/details-preferences.blp

```diff
@@ -335,66 +301,15 @@ template DetailsPreferences : .AdwPreferencesPage {
       activatable-widget: "switch_steam_runtime";
       title: _("Steam Runtime");
       subtitle: _("Provide a bundle of extra libraries for more compatibility with Steam games. Disable if you run into issues.");
-
+      visible: false;
       Switch switch_steam_runtime {
         valign: center;
       }
     }
 
-    .AdwActionRow {
-      activatable-widget: "switch_gamemode";
-      title: _("Feral GameMode");
-      subtitle: _("Apply a set of optimizations to your device. Can improve game performance.");
-
-      Switch switch_gamemode {
-        valign: center;
-      }
-    }
-
-    .AdwActionRow {
-      activatable-widget: "switch_gamescope";
-      title: "Gamescope";
-      subtitle: _("Manage how games should be displayed on the screen using Gamescope.");
-
-      Button btn_manage_gamescope {
-        tooltip-text: _("Manage Gamescope Settings");
-        valign: center;
-        icon-name: "applications-system-symbolic";
-
-        styles [
-          "flat",
-        ]
-      }
-
-      Switch switch_gamescope {
-        valign: center;
-      }
-    }
-
-    .AdwActionRow {
-      activatable-widget: "switch_vmtouch";
-      title: _("Preload Game Files");
-      subtitle: _("Improve loading time when launching the game multiple times. The game will take longer to start for the first time.");
-
-      Button btn_manage_vmtouch {
-        visible: false;
-        tooltip-text: _("Manage vmtouch Settings");
-        valign: center;
-        icon-name: "applications-system-symbolic";
-
-        styles [
-          "flat",
-        ]
-      }
-
-      Switch switch_vmtouch {
-        valign: center;
-      }
-    }
-
     .AdwActionRow row_cwd {
       title: _("Working Directory");
-
+      activatable-widget: btn_cwd;
```

Comment:
Adwaita Demo and Bottles don't do that. It's best to remove it.

```suggestion
```

Reply:
No, let's not - whichever other place should have an activatable-widget too. Besides, I'm not too sure how much the demo should be used for *design* reference.

Reply:
Sure. There's only one thing to discuss about.